src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import './Calculator.css';
+
+function Display() {
+  return (
+    <div className="DisplayArea">
+      <p>0</p>
+    </div>
+  );
+}
+
+function KeyArea() {
+  const Items = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+  return (
+    <div className="key-area">
+      {Items.map((Item, ind) => (
+        <button key={uuidv4()} id={`btn-${ind}`} type="submit" className="key-btn">{Item}</button>
+      ))}
+    </div>
+  );
+}
+
```

Comment:
- [ ] Good job creating these child components, however, we need to move these components to their respective new files. Kindly do that. Generally speaking, in React each component should have its own file for better code organization. You can export and them import them into this calculator component.

Reply:
@TedLivist Thanks for the insight, though the exercise didn't seem like it required the separation of the child components in different files. I will still go ahead and adjust as recommended. Thanks